pylint/constants.py

```diff
@@ -8,11 +8,6 @@
 
 from pylint.__pkginfo__ import version as pylint_version
 
-# Allow stopping after the first semicolon/hash encountered,
-# so that an option can be continued with the reasons
-# why it is active or disabled.
-OPTION_RGX = re.compile(r"\s*#.*\bpylint:\s*([^;#]+)[;#]{0,1}")
```

Comment:
The `re` module is then useless here. It's the reason the CI doesn't success.

Reply:
@wtracy thanks for this PR. I'am ok with it as far as the CI finishes successfully.